### DISCUSSION TITLE
Regular Channelopenings which fail from 2 specific nodes

### DISCUSSION THREAD

**Discussion by ziggie1984** [lightningnetwork/lnd#7277] [General]
I was monitoring my logs and saw 2 peers frequently trying to open a channel to my node but would always fail immedialty after the initial request. Also other noderunners are seeing these requests. They have a special signature:
``` 
channel type: private
capacity: 500_000_000 sats 
```

Are these some testnodes?

The nodes in question are:

```
02a46393dde952f46c703d978f9e4279508d7e35d5b54bf9e6d738565d513dba97
03d7ce4a11a52e19d987076ffa708b26212422f2a44f1de82c6a3b805ba5e542bb
```

They also want to connect to my node but without some features:

```
PEER: Peer(03d7ce4a11a52e19d987076ffa708b26212422f2a44f1de82c6a3b805ba5e542bb): disconnecting 03d7ce4a11a52e19d987076ffa708b26212422f2a44f1de82c6a3b805ba5e542bb@3.211.186.146:22028, reason: u
nable to start peer: data loss protection required
```

```
 PEER: Peer(03d7ce4a11a52e19d987076ffa708b26212422f2a44f1de82c6a3b805ba5e542bb): disconnecting 03d7ce4a11a52e19d987076ffa708b26212422f2a44f1de82c6a3b805ba5e542bb@3.211.186.146:25652, reason: u
nable to start peer: invalid remote features: missing feature dependency: 9
```

The last one I don't understand, because my node does not require feature 9, but fails the connection.

Thanks for your thoughts in Advance.

**Comment by zx9r** [comment]
I have exactly that same logs from that same 2 peers

**Comment by M1ch43lV** [comment]
My answer to these nodescanners:
```sudo ufw deny from 3.233.3.134``` for pubkey 02a46393dde952f46c703d978f9e4279508d7e35d5b54bf9e6d738565d513dba97
```sudo ufw deny from 3.211.186.146``` for pubkey 03d7ce4a11a52e19d987076ffa708b26212422f2a44f1de82c6a3b805ba5e542bb
